### PR TITLE
Added new cron capability

### PR DESCRIPTION
**Pull request by brianwilkinson**
Required for cron capability in MCH

### CODE REVIEW

File: files/hooks/entrypoint-pre.d/20_customerssh_config [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 export mUID=`id -u`
+export mGID=65534
```
Comment (leroyshirtoFH) [inline files/hooks/entrypoint-pre.d/20_customerssh_config]: Is this supposed to have a fixed GID or is this left over from testing?

Comment (brianwilkinson) [inline files/hooks/entrypoint-pre.d/20_customerssh_config]: That is deliberate. If there is no default value then uGID usually doesn't get set and the consequence is that when you log in it's as an unknown user. This breaks cron.

File: Dockerfile [hunk 2]
```diff
@@ -36,9 +36,29 @@ RUN \
...
     HOME=/var/www
 
 WORKDIR /var/www
+
+# Install and configure the cron service
+ENV EDITOR=/usr/bin/vim \
+	CRON_LOG_FILE=/var/spool/cron/cron.log \
+	CRON_LOCK_FILE=/var/spool/cron/cron.lock \
+	CRON_ARGS=""
+RUN \
```
Comment (astrolox) [inline Dockerfile]: Combine in to a single RUN command to reduce the number of layers, unless we have a reason not to.

File: files/init/run_forever.sh [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+bash /init/entrypoint run
+sleep infinity
```
Comment (astrolox) [inline files/init/run_forever.sh]: Is this the /bin/sleep binary?
If so add  exec to the start so the bash instance is removed from memory.

**Comment by leroyshirtoFH** [comment]
lgtm

**Comment by astrolox** [comment]
Closing this pull request

**Pull request closed without merging**